File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1052.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        int n = ler.nextInt();

        String[] meses = {"January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"};        

        System.out.println(meses[n-1]);
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1052.kt
import java.util.*
import java.lang.Math

fun main(args: Array<String>) {
    val ler = Scanner(System.`in`)

    var n: Int = ler.nextInt();
    
    var meses = arrayOf("January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December");     
        
    System.out.println(meses[n-1]);
}